docs/use_cases/met_tool_wrapper/ExtractTiles/ExtractTiles_mtd.py

```diff
@@ -0,0 +1,122 @@
+"""
+ExtractTiles: MTD Input
+=======================
+
+met_tool_wrapper/ExtractTiles/ExtractTiles_mtd.conf
+
+"""
+##############################################################################
+# Scientific Objective
+# --------------------
+#
+# Read a MODE Time Domain (MTD) output file and use the centroid latitude
+# and longitude values of the object pairs to
```

Comment:
Maybe we could add "MTD cluster" ahead of "object pairs", and then refer to the MET MTD output docs here:
https://met.readthedocs.io/en/latest/Users_Guide/mode-td.html#mtd-output

I can make these changes if you want.